nginx_stage/lib/nginx_stage/generators/nginx_clean_generator.rb

```diff
@@ -111,7 +111,7 @@ class NginxCleanGenerator < Generator
             FileUtils.rmdir(dir)
           end
         rescue StandardError => e
-          warn "Error trying to clean up the PID file directory of disabled user #{u}: #{e.message}"
+          warn "Error trying to clean up the PID file #{dir} of disabled user: #{e.message}"
```

Comment:
I think the word "directory" should be retained, but otherwise this is okay.

I.e. `"Error trying to clean up the PID file directory #{dir} of disabled user: #{e.message}"`